### PR TITLE
Skip creating table & spawing thread if update_check_interval=0

### PR DESCRIPTION
**Pull request by kaxil**
If `[astronomer] update_check_interval=0` then we should skip creting table or creating threads to get new versions

### CODE REVIEW

**Review by ashb — APPROVED** [review]

File: astronomer/airflow/version_check/plugin.py [hunk 1]
```diff
@@ -32,6 +33,11 @@ def on_load(cls, *args, **kwargs):
         # Hook in to various places in Airflow in a slightly horrible way -- by
         # using functools.wraps and replacing the function.
 
+        if conf.getint("astronomer", "update_check_interval", fallback=24 * 60 * 60) == "0":
+            log.debug(
+                "Skipping running update_check_plugin as [astronomer] update_check_interval = 0")
```
Comment (ashb) [inline astronomer/airflow/version_check/plugin.py:38]: We should also not register the blueprint in the UI if this config is 0

**Review by ashb — CHANGES_REQUESTED** [review]

File: astronomer/airflow/version_check/plugin.py (hunk 1 above)
Comment (kaxil) [reply astronomer/airflow/version_check/plugin.py:38]: Will do

Comment (kaxil) [reply astronomer/airflow/version_check/plugin.py]: Done @ashb

**Comment by kaxil** [comment]
This depends on https://github.com/astronomer/astronomer-airflow-version-check/pull/19

**Review by ashb — APPROVED** [review]

**Pull request merged**